### PR TITLE
MEPTS-769 | Changes on compositionStrings and queries in cat.11

### PR DESCRIPTION
**Pull request by rupansana12**
change to compositionStrings and queries in cat.11

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java [hunk 1]
```diff
@@ -1225,4 +1225,81 @@ public static CohortDefinition getMQ11DenB4(
...
+            + "             INNER JOIN obs o ON e.encounter_id = o.encounter_id   "
+            + "             WHERE   p.voided = 0   "
+            + "                     AND e.voided = 0   "
+            + "                     AND o.voided = 0   "
+            + "                     AND e.encounter_type = ${6}    "
+            + "                     AND o.concept_id = ${6332}    "
+            + "                     AND o.value_coded = ${1065}    "
+            + "                     AND e.encounter_datetime BETWEEN :startDate AND :endDate   "
+            + "                     AND e.location_id = :location    "
+            + "             GROUP BY p.patient_id  "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: @rupansana12 I cant see I thing that refers to female patient
@Edgencio I guess you helped to build this query

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@rupansana12 @Edgencio

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java [hunk 2]
```diff
@@ -1225,4 +1225,83 @@ public static CohortDefinition getMQ11DenB4(
...
+            + "             WHERE   p.voided = 0   "
+            + "                     AND e.voided = 0   "
+            + "                     AND o.voided = 0   "
+            + "                     AND e.encounter_type = ${6}    "
+            + "                     AND o.concept_id = ${6332}    "
+            + "                     AND o.value_coded = ${1065}    "
+            + "                     AND e.encounter_datetime BETWEEN :startDate AND :endDate   "
+            + "                     AND e.location_id = :location    "
+            + "                     AND per.gender = 'F'      "
+            + "             GROUP BY p.patient_id  "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: @rupansana12 this group by will give you some wrong results
@Edgencio @helderjosue

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
This group by will give you wrong results

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java (hunk 2 above)
Comment (rupansana12) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: @psmatsinhe  sure, do you recommend removing it?

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**